schedule/yast/repo_inst.yaml

```diff
@@ -10,28 +10,17 @@ vars:
   VALIDATE_INST_SRC: '1'
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/validation/repo_inst
-  - installation/validation/validate_install_repo
-  - installation/access_beta_distribution
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/register_module_desktop
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_SLES_with_GNOME
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/grub_test
-  - installation/first_boot
-  - console/validate_mirror_repos
+  product_selection:
+    - installation/validation/repo_inst
+    - installation/validation/validate_install_repo
+    - installation/product_selection/install_SLES
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  add_on_product:
+    - installation/add_on_product/skip_install_addons
```

Comment:
In the adaption in [PR#16218](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16218) this will be the default already.

Reply:
OK